tiledb/sm/cpp_api/array_experimental.h

```diff
@@ -68,6 +68,18 @@ class ArrayExperimental {
     ctx.handle_error(
         tiledb_array_load_all_enumerations(ctx.ptr().get(), array.ptr().get()));
   }
+
+  /**
+   * Load all enumerations for all schemas in the array.
+   *
+   * @param ctx The context to use.
+   * @param array The array to load enumerations for.
+   */
+  static void load_enumerations_all_schemas(
```

Comment:
should this be experimental for some reason?

Reply:
I put this in experimental because the other enumerations APIs were still experimental and it felt weird to add a new API that was so similar outside of experimental